docs/docs/node-exemption

```diff
@@ -0,0 +1,22 @@
+---
+title: Node Exemption
+---
+
+## Exempting Nodes from the Eraser Pipeline
+
+When deploying Eraser, you can specify whether there is a list of nodes you would like to `include` or `exclude` from the cleanup process using the `--filter-nodes` argument.  
+
+```bash
+$ helm install eraser-test manifest_staging/charts/eraser --create-namespace --namespace=eraser-system --set controllerManager.image.AdditionalArgs={--filter-nodes=include}
```

Comment:
we don't need to mention helm install or manifest_staging, we can just mention it's available in chart